src/Context.cpp

```diff
@@ -218,7 +218,7 @@ void Context::InitControlDeck(std::vector<CONTROLLERBUTTONS_T> additionalBitmask
         return;
     }
 
-    mControlDeck = std::make_shared<ControlDeck>(additionalBitmasks);
+    mControlDeck = std::make_shared<LUS::ControlDeck>(additionalBitmasks);
```

Comment:
reason this is in draft

Reply:
Hmm. What is the additional bitmasks doing again?

We could push this to the derived class. If we do that, we want some kind of internal overload able init function that's call from the main one.

Reply:
we don't have a derived context

Reply:
I meant for the ControlDeck class.

Reply:
even without `additionalBitmasks` we can't make an instance of an abstract class
```
error: invalid new-expression of abstract class type ‘Ship::ControlDeck’
```